documentation_builder/metadata.ipynb

```diff
@@ -0,0 +1,548 @@
+{
+ "nbformat": 4,
+ "nbformat_minor": 0,
+ "metadata": {
+  "kernelspec": {
+   "display_name": "Python 3",
+   "language": "python",
+   "name": "python3"
+  },
+  "language_info": {
+   "codemirror_mode": {
+    "name": "ipython",
+    "version": 3
+   },
+   "file_extension": ".py",
+   "mimetype": "text/x-python",
+   "name": "python",
+   "nbconvert_exporter": "python",
+   "pygments_lexer": "ipython3",
+   "version": "3.6.0"
+  },
+  "colab": {
+   "name": "metadata.ipynb",
+   "provenance": []
+  }
+ },
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "bVgYSbcPLbwU",
+    "colab_type": "text"
+   },
+   "source": [
+    "# Metadata\n",
+    "\n",
+    "Metadata in the form of annotations, provenance or additional key:value pairs is an important asset for a model.\n",
```

Comment:
This notebook sounds it is written for people designing standards. Will a newcomer to metabolic modeling know what provenance means? This should become more accessible.

Reply:
I agree. This was more for testing/low level documentation.